### ISSUE TITLE
Paused Mode Downloads

### ISSUE THREAD

**Issue by lilpea** [nicotine-plus/nicotine-plus#2592]
As a [thread on r/Soulseek](https://old.reddit.com/r/Soulseek/comments/15b1g11/adding_in_paused_mode_in_nicotine/) pointed out, it would be useful to add a folder to the downloads list, having it immediately be in paused mode. This same feature is also supported in the SoulseekQt client, which is another reason for adding it.

**Comment by slook** [comment]
Closing in favour of https://github.com/nicotine-plus/nicotine-plus/issues/1659

**Comment by mathiascode** [comment]
> Closing in favour of #1659

Note that this functionality is also needed when browsing a user, so we either need to implement the download dialog there as well, or come up with another solution.

**Comment by mathiascode** [comment]
@lilpea Would a maximum number of download slots (sequential downloads) be an alternative? https://github.com/nicotine-plus/nicotine-plus/issues/110

I'd assume the purpose of adding paused downloads is to avoid starting all of them at once?

**Comment by slook** [comment]
Yes, the purposes are both to avoid local network performance bottle-necking and also to evade a peer's user count/size limiter.

However, as modern clients become more efficient at handling larger queues, then manually managing the local queue becomes less important.
